astropy/utils/metadata/__init__.py

```diff
@@ -0,0 +1,34 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""This module contains helper functions and classes for handling metadata."""
+
+from .core import MetaAttribute, MetaData
```

Comment:
Maybe follow current standard and `import *` from those (defining `__all__`)

Reply:
Done.